packages/client-features/src/utils.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Helper function to check if we're in the post editor
+ */
+export const isInPostEditor = (): boolean => {
+	try {
+		return !! document.querySelector( '.block-editor' );
```

Comment:
Better to check for `wp.data.select('core/editor')` and that we are not in the site editor, which also uses the editor store.